pontoon/settings/base.py

```diff
@@ -531,7 +531,7 @@ def _allowed_hosts():
 
     host = urlparse(settings.SITE_URL).netloc  # Remove protocol and path
     host = host.rsplit(':', 1)[0]  # Remove port
-    return [host]
+    return [host] + ['0.0.0.0']
```

Comment:
I'd restore the previous line, We don't intend to keep `0.0.0.0` hard-coded.

Reply:
These were for testing purpose and will surely remove them in subsequent PR